docs/pages/dsVideoDevice/ds-video-device_L3_TestProcedure.md

```diff
@@ -0,0 +1,368 @@
+
+- Execution process:
+
+  The test will:
+
+  - Download all the required artifacts and video streams.
+  - Copy them to the target directory.
+  - Automatically start the test execution.
+
+- Play the video stream:
+
+  Test starts the video stream playback specified in the test setup configuration file.
+
+- Iterating through supported zoom modes:
+
+  The test will repeat for different supported zoom modes. On setting each zoom mode user need to verify it is applied in video stream.
+
+- User interaction for verification:
+
+  For each iteration:
+
+  - The test will ask: "Is the selected Zoom mode applied in the device's output? (Y/N):"
+
+- Completion and results:
+
+  Once all necessary user actions are completed, the test will evaluate the results and display whether the test Passed or Failed.
+
+### dsVideoDevice_test3_SetDisplayFramerate.py
+
+#### Platform Supported - test03
+
+- Sink
+
+#### User Input Required - test03
+
+Yes (This will be automated later).
+
+#### Acceptance Criteria - test03
+
+Verify the impact of the supported display framerates for **Stream #2**.
+
+#### Expected Results - test03
+
+This test will evaluate the supported display framerates. The user should notice a video playback when display framerates are applied.
+
+#### Test Steps - test03
+
+- Select the test file:  
+
+  - Run the Python script **`dsVideoDevice_test3_SetDisplayFramerate.py`**
+
+- Execution process:
+
+  The test will:
+
+  - Download all required artifacts and video streams.
+  - Copy them to the target directory.
+  - Automatically begin the test execution.
+
+- Stream Playback:
+
+  - The test will play the designated video stream.
+  - During playback, the test will prompt the user to verify whether displayRate with a streamRate acceptable.
+
+- Iterating through supported display framerates:
+
+  The test will repeat for different supported framerates. Before that set the FRF mode to `Disable`. And, then on setting each displayRate user need to verify it is applied in video stream.
+
+- User Interaction:
+
+  For each prompt, the user should assess the video playback output and respond:
+
+  - Is a displayRate with a streamRate acceptable? (Y/N):, the user should press **Y** to confirm (this will pass the step).
+  - If not, press **N** (this will fail the step).
+
+- Completion and results:
+
+  After receiving all user inputs, the test will conclude and provide a final result (PASS/FAIL).
+
+### dsVideoDevice_test4_SetAndGetFRFMode.py
+
+#### Platform Supported - test04
+
+- Sink
+
+#### User Input Required - test04
+
+Yes (This will be automated later).
+
+#### Acceptance Criteria - test04
+
+Verify the video playback with FRF mode enabled/disabled for all supported video device for **Stream #1**.
+
+#### Expected Results - test04
+
+The test evaluates the effect of FRF mode on video playback. The user should notice a clear difference in video output when FRF is toggled between Enable and Disable.
```

Comment:
The document should specify the expected display modes or configurations for each corresponding frame rate, outlining the criteria for successful verification.

Reply:
updated